internal/controller/services/monitoring/monitoring_controller_support.go

```diff
@@ -36,6 +37,7 @@ func getTemplateData(ctx context.Context, rr *odhtypes.ReconciliationRequest) (m
 
 	templateData["Traces"] = monitoring.Spec.Traces != nil
 	templateData["Metrics"] = monitoring.Spec.Metrics != nil
+	templateData["GPUMetrics"] = rr.DSCI.Spec.Monitoring.ManagementState == operatorv1.Managed && monitoring.Spec.Metrics != nil
```

Comment:
Is the `ManagementState` check required ? To my (limited) understanding the `Monitoring` CR is created only if `ManagementState == operatorv1.Managed` or for the RHOAI Managed case (but I could be wrong)

Reply:
youre correct it can be simplified, thanks